src/com/haskforce/refactoring/HaskellRefactoringSupportProvider.java

```diff
@@ -13,4 +13,6 @@
     public boolean isMemberInplaceRenameAvailable(@NotNull PsiElement element, PsiElement context) {
         return element instanceof HaskellNamedElement;
     }
+
+
```

Comment:
Could we remove the extra spacing?  Especially since nothing was changed in this file.